app/javascript/src/components/editor/Modals/TranslationKeysModal.svelte

```diff
@@ -76,6 +183,11 @@
         {/if}
 
         <button on:click={addKey} class="button button--small button--square w-100 mt-1/8">Create</button>
+
+        <div class="button-group mt-1/4">
+          <button on:click={() => importKeysFromClipboard()} class="button button--small button--secondary button--square w-100" data-tooltip="From CSV/TSV in your clipboard">Import</button>
+          <button on:click={() => exportKeys()} class="button button--small button--secondary button--square w-100" data-tooltip="To TSV (for Excel)">Export</button>
+        </div>
```

Comment:
To clean thing up a bit, I think it makes more sense to move this and the new function above to a new Svelte component.

Reply:
Agreed!